docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -440,6 +440,8 @@ kubectl apply --namespace <NAMESPACE_NAME> -f identity-connectors-secret.yaml
 
 Camunda v8.2 uses Keycloak v19 which depends on PostgreSQL v15. That is a major change for the dependencies. Currently there are two recommended options to upgrade from Camunda 8.1.x to 8.2.x:
 
+<!--- Should this be updated to Keycloak v21? --->
```

Comment:
@aabouzaid Mind having a look at this lingering comment?

Reply:
@christinaausley what is the problem here? :thinking: